### ISSUE TITLE
Create UI for move VR cam to edit cam position

### ISSUE THREAD

**Issue by mattatgit** [vizorvr/vizor#859]
Button for centre VR cam at edit cam position

**Comment by mattatgit** [comment]
![zero cam](https://cloud.githubusercontent.com/assets/6858525/11385915/89c6fc08-9371-11e5-9b5d-012b2e7d9243.png)
Sketch file ref Vizor Navbar States 151125

**Comment by mattatgit** [comment]
Ping @kkananen @gmarinov

**Comment by fthr** [comment]
This has been discussed recently so moving it up. Hotkey is now in place, should create the UI button for it too, right?

**Comment by mattatgit** [comment]
Yeah i think the only blocker here is: what to call it ? @kkananen is against Zero Cam, and i see his point. We're not putting the VR camera at 0,0,0, but at 0,0,0 _relative_ to the current edit cam position. Sooo you _could_ argue that Zero Cam is ok if you interpret it that way. If so, we can just roll this out. @fthr

**Comment by mattatgit** [comment]
Here's what we could add for the Cam placement mechanism using "=". This seems reasonably clear to me, and better than "zero cam"

![place_cam](https://cloud.githubusercontent.com/assets/6858525/16477635/918a21d4-3ed6-11e6-862e-b5bfa12626f1.png)
